proto/ptf/base_test.py

```diff
@@ -49,35 +50,30 @@ def __get__(self, instance, owner):
                        *(self.args or ()), **(self.keywords or {}))
 
 # Convert integer (with length) to binary byte string
-# Equivalent to Python 3.2 int.to_bytes
-# See
-# https://stackoverflow.com/questions/16022556/has-python-3-to-bytes-been-back-ported-to-python-2-7
-# TODO: When P4Runtime implementation is ready for it, use
-# minimum-length byte sequences to represent integers.  For unsigned
-# integers, this should only require removing the zfill() call below.
-def stringify(n, length):
+def stringify(n, length=None):
     """Take a non-negative integer 'n' as the first parameter, and a
     non-negative integer 'length' in units of _bytes_ as the second
     parameter.  Return a string with binary contents expected by the
     Python P4Runtime client operations.  If 'n' does not fit in
     'length' bytes, it is represented in the fewest number of bytes it
     does fit into without loss of precision.  It always returns a
     string at least one byte long, even if value=width=0."""
```

Comment:
can you update this function comment to indicate what it means when `length=None`?